extensions/sql-database-projects/src/test/baselines/openSqlProjectBaseline.xml

```diff
@@ -94,6 +94,14 @@
       <DatabaseVariableLiteralValue>master</DatabaseVariableLiteralValue>
     </ArtifactReference>
   </ItemGroup>
+  <ItemGroup>
+    <PreDeploy Include="Script.PreDeployment1.sql"/>
+    <None Include="Script.PreDeployment2.sql"/>
+  </ItemGroup>
+  <ItemGroup>
+    <PostDeploy Include="Script.PostDeployment1.sql"/>
+    <None Include="Tables\Script.PostDeployment1.sql"/>
```

Comment:
Do you know how these None entries get used?  If they're just to establish relative path for the Pre/Post deploy scripts, I wonder why they're saved as separate XML instead of just putting the path in the Pre/PostDeploy entry...

Reply:
So, for a sqlproj only one pre and one post deployment script can be added. All the additional scripts get added as None entries. Also for non-build scripts, they get added as None. These "None" scripts are present in the project but not used during Build or Deploy. If I am not wrong, "Exclude from project" changes tags from Build/PreDeploy/PostDeploy to None.